src/cobertura.rs

```diff
@@ -3,14 +3,15 @@ use quick_xml::{
     Writer,
 };
 use rustc_hash::FxHashMap;
-use std::io::{BufWriter, Cursor, Write};
 use std::path::Path;
 use std::time::{SystemTime, UNIX_EPOCH};
+use std::{
+    io::{BufWriter, Cursor, Write},
+};
```

Comment:
Could you undo these unrelated formatting changes?

Reply:
pushed